MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -11,6 +11,11 @@ static partial class GamePad
     {
         internal static bool Back;
 
+        private static int PlatformGetMaxIndex()
+        {
+            return 16;
```

Comment:
As bellow, it would cause a crash to code like:
```c#
GamePad.GetState(PlayerIndex.Two)
```
when one gamepad is connected.